### DISCUSSION TITLE
Version for Linux

### DISCUSSION THREAD

**Discussion by gladmaxt** [Rem0o/FanControl.Releases#444] [Ideas]
Would you plan version for Linux, like Ubuntu?

**Comment by Rem0o** [comment]
Don't think the backend is Linux compatible for controlling fans. If so, then would need to upgrade to .NET 6, which is not a given also.

**Reply by Cazerus1** [reply]
that is such a shame, the only reason I dont run linux full time on my pc is because I cant control the fan speeds. I have 10 fans in total running through Icue and armoury crate, which works well in windows, though I am going to run your fancontrol with openrgb to now run these in windows. The fan control options available for Linux, just dont work particularly well so the pc is too noisy. Your fancontrol software, as Jay says is a fantastic piece of work, many thanks to you for your hard work.

**Reply by jaroslavstreit** [reply]
Yep, same for me, unfortunately seems I'll need to use Ubuntu (because it's Artificial Intelligence workstation). So only option is lm-sensors and fancotrol, described here: https://askubuntu.com/questions/22108/how-to-control-fan-speed
